www/src/components/docs-breadcrumb.js

```diff
@@ -0,0 +1,101 @@
+import React from "react"
+import { Link } from "gatsby"
+import ChevronRight from "react-icons/lib/md/chevron-right"
+import ChevronLeft from "react-icons/lib/md/chevron-left"
+import getActiveItem from "../utils/sidebar/get-active-item"
+import getActiveItemParents from "../utils/sidebar/get-active-item-parents"
+import { mediaQueries, space, fontSizes } from "../utils/presets"
+
+const Separator = ({ character = <ChevronRight /> }) => (
+  <span style={{ margin: `0px ${space[1]}` }} role="presentation">
+    {character}
+  </span>
+)
+
+const BreadcrumbNav = ({ children, mobile = false }) => (
+  <nav
+    aria-label="breadcrumb"
+    css={{
+      fontSize: fontSizes[1],
+      display: `${mobile ? `inherit` : `none`}`,
+      [mediaQueries.md]: {
+        display: `${mobile ? `none` : `inherit`}`,
+      },
+      marginBottom: space[2],
+    }}
+  >
+    {children}
+  </nav>
+)
+
+const Breadcrumb = ({ itemList, location }) => {
+  const activeItem = getActiveItem(itemList.items, location, undefined)
+  const activeItemParents = getActiveItemParents(itemList.items, activeItem, [])
+  const topLevel = itemList.key
+  const topLevelTitle = itemList.breadcrumbTitle || itemList.title
+  // return a shorter version of the breadcrumb on the intro page
+  // because the docs intro page isn't generated from markdown
+  if (activeItem.title === `Introduction`) {
+    return (
+      <>
+        {/* only the breadcrumb nav of the proper viewport is displayed */}
+        <BreadcrumbNav>
+          <Link to="/">Home</Link>
+          <Separator />
+          {topLevelTitle}
+        </BreadcrumbNav>
+        <BreadcrumbNav mobile>
+          <Separator character="<" />
```

Comment:
should it not be the ChevronLeft here also?

```suggestion
          <Separator character={<ChevronLeft />} />
```

Reply:
Yeah that's a good catch! Wanna add that into your PR in #15643?

Reply:
ok - included in PR #15643